#========================================================================
#  This software is free: you can redistribute it and/or modify
#  it under the terms of the GNU Lesser General Public License Version 3,
#  as published by the Free Software Foundation.
#
#  This software is distributed in the hope that it will be useful,
#  but WITHOUT ANY WARRANTY; without even the implied warranty of
#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
#  GNU Lesser General Public License for more details.
#
#  You should have received a copy of the GNU Lesser General Public License
#  Version 3 in the file COPYING that came with this distribution.
#  If not, see <http:#www.gnu.org/licenses/>.
#========================================================================
#!
 # \file    gpsDrift.srv
 # \brief   gps drift service
 # \author  Fatma Faruq 
 # \other   HACMS CMU Advisor Manuela Veloso
 #
##========================================================================
# GPS Drift Service 

float32 drift_x 
float32 drift_y 
float32 drift_z
float32 drift_time 
---
int8 err_code 

